src/azure-cli-core/azure/cli/core/tests/test_application.py

```diff
@@ -54,7 +54,7 @@ def handler(**kwargs):
         def other_handler(**kwargs):
             self.assertEqual(kwargs['args'], 'secret sauce')
 
-        app = Application()
+        app = AZ_CLI
```

Comment:
The aim was to remove a global Application.
Should be able to create a mock CLI like this https://github.com/Microsoft/knack/blob/master/tests/util.py